daprdocs/content/en/operations/monitoring/newrelic.md

```diff
@@ -0,0 +1,347 @@
+---
+type: docs
+title: "How-To: Set-up New Relic to collect and observe metrics, traces and logs from Dapr and the underlying applications"
```

Comment:
I suggest to simplify the title to "How-To: Set-up New Relic for Dapr observability"

Reply:
agree